### DISCUSSION TITLE
Specify google drive folder path?

### DISCUSSION THREAD

**Discussion by mvevitsis** [floccusaddon/floccus#1636] [Q&A]
How can I set this up so that my bookmarks files aren't in the root of my Google drive? 

I want it to be something like /Floccus/bookmarks.xbel

Is this possible?

**Comment by Ciberyan** [comment]
I second this question !
I try many options but nothing works
Thanks in advance

**Comment by marcelklehr** [comment]
Currently paths are not supported but you can move the file to the desired location after floccus has created it (without changing anything in the floccus settings).

**Comment by Ciberyan** [comment]
Hello Marcel
I try that but floccus doesn't update it after... Probably because it
doesn't know anymore where the file is located...

Le jeu. 31 oct. 2024, 11:41, Marcel Klehr ***@***.***> a
écrit :

**Reply by jhgit** [reply]
> Hello Marcel I try that but floccus doesn't update it after... Probably because it doesn't know anymore where the file is located...

Using firefox (132.0.2), I used floccus to create a bookmark file on my google drive.  It created a file at the top level.  When I used the google drive web interface to move that file to a folder a created (two levels down from the top level), I was still able to sync to it.  And on a separate browser on a separate computer, I imported the floccus profile and was able to read in floccus bookmarks there.

So moving the file (to a different folder in google drive) "worked" for me.  I did not try _copying_ the bookmark file to a different folder.

**Comment by jhgit** [comment]
Would it be hard to allow the full URL that you can get from "Copy Link" in Google Drive (https://drive.google.com/...)?  Then one could create a folder and an empty .xbel file on Google Drive where you want the floccus bookmarks file and use "Copy Link" to add that when adding a profile in floccus.

**Reply by marcelklehr** [reply]
Ideally there should be some sort of folder picker that shows your google drive, but I haven't found the energy to invest time into this so far. If you want, you can have a go at it, I'm happy to provide pointers and help if you get stuck :)
